src/Symfony/Component/Workflow/DefinitionBuilder.php

```diff
@@ -54,20 +54,36 @@ public function clear()
     {
         $this->places = [];
         $this->transitions = [];
-        $this->initialPlace = null;
+        $this->initialPlaces = null;
         $this->metadataStore = null;
 
         return $this;
     }
 
     /**
+     * @deprecated since Symfony 4.3. Use the setInitialPlaces() instead.
```

Comment:
the ... method, or "the" should be removed

Reply:
Fixed. And also `Definition::getInitialPlace()`